### ISSUE TITLE
Missing Subscribe/Unsubscribe

### ISSUE THREAD

**Issue by muuvmuuv** [GitHawkApp/GitHawk#2149]
I‘m missing a button to subscribe and unsubscribe a issue or pr. So this is a feature request :)

<details>
<summary>Bug Report Dump (Auto-generated)</summary>
<pre>
Version 1.22.0 (1534395229)
Device: iPhone 6s (iOS 11.4.1)
TestFlight: false
</pre>
</details>

**Comment by BasThomas** [comment]
You can subscribe/unsubscribe to an issue or pull request in your inbox. These should have this “loudspeaker” icon. 

Where else would you expect to see this? Would you like to be able to subscribe to an issue you’re not subscribed to in the first place?

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by muuvmuuv** [comment]
Yes, exactly. The loudspeaker icon is new, isn't it? Will it subscribe only for app notification or for GitHub, so that I can view notification is the browser too?

**Comment by Huddie** [comment]
The loudspeaker 📢 is an interesting “shortcut”
You can only see it in your inbox so it means your already subscribed to it.
If you unsubscribe it gives you the chance to subscribe assuming you don’t mark it read, meaning you don’t read the issue. 
I feel more likely you read the issue and then want to unsubscribe which means you must go to view all and then unsubscribe? It just feels a bit weird. 

I’d suggest removing the loudspeaker 📢 from inbox and moving it into the manager button below the divider along with #2140.

I think then inbox cells can be simplified to just the read button and a more button see #2127 for an example. 


<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by rnystrom** [comment]
Sub/un in the burger menu would be great! Also removes the need to hide/show the menu button which would clean up a lot of code!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ijm8710** [comment]
Would love this as well

**Comment by Huddie** [comment]
@ijm8710 I actually meant to tag this issue in my PR, the PR you tagged is to add this support.

**Comment by ijm8710** [comment]
Sweet, would it make sense to have a reference section where it says you’re actively watching/subscribed to said thread like it does on github. Somewhat similar to how you guys created the reference subsections

**Comment by Huddie** [comment]
Not sure I follow.
In the manger content menu it says either : subscribe, unsubscribe so you’ll know your current status 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by ijm8710** [comment]
[Meant this](https://i.imgur.com/GMEWjdG.jpg)

While what you said is absolutely true, I just meant having the blurb so you know without avruoning the manager button

**Comment by Huddie** [comment]
@ijm8710 maybe for a separate PR, focusing on this one first. Good suggestion just wanna do one thing at a time 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by goerz** [comment]
Strongly seconded. I very frequently subscribe to issues on repos where I'm "just a user/beta tester", so that I get notified when a fix for a particular problem is available. This very issue would a prime example! (of course, now I've commented on it, which subscribes me to updates, but in most cases, I'm just lurking without a comment).
